src/docs/common/callouts/callout-spring-ann-service.adoc

```diff
@@ -0,0 +1 @@
+https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Service.html[org.springframework.stereotype.Service] maps to https://docs.micronaut.io/latest/api/io/micronaut/context/annotation/Bean.html[io.micronaut.context.annotation.Bean] at compilation time. A Spring's `@Service` indicates that they hold business logic.
```

Comment:
I'm not sure which of these is more accurate, but it should be one of these:

```suggestion
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Service.html[org.springframework.stereotype.Service] maps to https://docs.micronaut.io/latest/api/io/micronaut/context/annotation/Bean.html[io.micronaut.context.annotation.Bean] at compilation time. Spring's `@Service` indicates that they hold business logic.
```
OR

```suggestion
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Service.html[org.springframework.stereotype.Service] maps to https://docs.micronaut.io/latest/api/io/micronaut/context/annotation/Bean.html[io.micronaut.context.annotation.Bean] at compilation time. A Spring `@Service` indicates that they hold business logic.
```

Also, I'm not sure what "they" refers to in that last sentence. Should that be clarified?